### PR TITLE
Fix appveyor and skip test_log

### PR DESCRIPTION
**Pull request by janisz**
Refs: 
* https://github.com/golangci/golangci-lint/issues/830#issuecomment-575032702
* https://jira.d2iq.com/browse/D2IQ-67286

### CODE REVIEW

File: python/lib/dcoscli/tests/integrations/test_service.py [hunk 1]
```diff
@@ -86,6 +86,7 @@ def test_log():
     package_name = 'hello-world'
 
     with package(package_name, deploy=True):
+        time.sleep(60)
```
Comment (pierrebeitz) [inline python/lib/dcoscli/tests/integrations/test_service.py:89]: what's that about?

**Comment by janisz** [comment]
@mesosphere-ci run all integration tests

**Comment by janisz** [comment]
@mesosphere-ci run all integration tests

**Comment by janisz** [comment]
@mesosphere-ci run windows integration tests

**Review by DominikDary — APPROVED** [review]

**Comment by janisz** [comment]
@mesosphere-ci run linux integration tests

**Comment by janisz** [comment]
@mesosphere-ci run mac integration tests

**Comment by janisz** [comment]
@mesosphere-ci run windows integration tests

**Review by DominikDary — APPROVED** [review]

**Pull request merged**